### ISSUE TITLE
As a collector, I would like to batch load CollectingEvents from an excel sheet

### ISSUE THREAD

**Issue by LordFlashmeow** [SpeciesFileGroup/taxonworks#2429]
(Speaking for @bpescador on this one)

A core part of the workflow is creating CE right after returning from a trip. It would be a huge boon to be able to import them all at once from an excel sheet. Ideally it would: 
* create the georeference if given coordinates

Not urgent, but a good use for the new importer framework when it's ready.

**Comment by mjy** [comment]
To me all but `add collectors` is done in DwC importer, no?  

Ideally they would use the spreadsheet, stubbed out for them, then we just add ability to add all collectors to each record if they are the same?

**Comment by LordFlashmeow** [comment]
DwC importer is focused on occurrences (or taxa). Would it be simpler to make it a separate task or create a new mode for the importer?

**Comment by mjy** [comment]
@LordFlashmeow right, sorry for my confusing the issue.  I think if we abstract what we have in DwC we have the basis.  I'm going to re-categorize this as a Task request rather than thinking of further loading the functionality/features onto DwC importer.

I would, however, also encourage you and @bpescador to consider what it would take to capture the data in a on/offline app in the field directly.  With clever presets you could do a lot of stubbing and only have to add minimal data in the field, then on (re)sync the collecting events would be loaded.  I.e. we could, maybe (I personally think for sure), do better than spreadsheets.

**Comment by debpaul** [comment]
@LordFlashmeow this is done now by you, yes? 
1. You use TripCode as your unique ID to prevent new CE being created on upload of new specimen records using DwC-A Importer?
2. You used "work-a-round" to add new CE using the DwC-A Occurrence Importer --- and then deleting the "Fake" Specimens.
Right?

I'm adding that we add use the TW generated CE Id (for a given set of Specimens to be imported), instead of the "Trip Code" (which not everyone uses). That way you could get the TW CE Ids out and put them into the form being prepared for upload of specimen records.
